lib/chef/provider/support/yum_repo.erb

```diff
@@ -1,4 +1,4 @@
-# This file was generated by Chef
+# This file was generated by Chef Infra
```

Comment:
Any reason this can't use the dist constants?

Reply:
Someone would need to do that.